Dockerfile

```diff
@@ -5,7 +5,7 @@ COPY . /workspace
 RUN make build-docker
 
 # Use distroless as final image
-FROM gcr.io/distroless/base-debian11
+FROM gcr.io/distroless/base-debian11@sha256:e5853c0285c4c07ab5724d0b582c9b168f6c8dfa330627d22f814d98d77c5b85
```

Comment:
This is pointing to a invalid image,  it's the .sig tag used for cosigning.

```bash
❯ docker pull gcr.io/distroless/base-debian11@sha256:e5853c0285c4c07ab5724d0b582c9b168f6c8dfa330627d22f814d98d77c5b85
gcr.io/distroless/base-debian11@sha256:e5853c0285c4c07ab5724d0b582c9b168f6c8dfa330627d22f814d98d77c5b85: Pulling from distroless/base-debian11
unsupported media type application/vnd.dev.cosign.simplesigning.v1+json
```

You would want to pull the SHA from the latest-amd64 tag, currently sha256:8e8769bf7b83830995154e6c37c7d0de27ed91e58eaf45662057eea4c22705eb

Reply:
I'm sorry, mistakes were made. I think the latest for latest-amd64 changed again, so I updated to the newest one: 
![image](https://user-images.githubusercontent.com/5712117/211083013-30afff0c-7ea3-4b6b-80a5-efc0f301411d.png)

Reply:
No problem, thanks for the PR!